packages/internal-workflow-common/src/errors.ts

```diff
@@ -36,11 +40,11 @@ export class WorkflowExecutionAlreadyStartedError extends Error {
 }
 
 /**
- * Thrown when workflow with the given id is not known to the Temporal service.
+ * Thrown when a Workflow with the given Id is not known to Temporal Server.
  * It could be because:
- * - ID passed is incorrect
- * - Workflow execution is complete (for some calls e.g. terminate),
- * - workflow was purged from the service after reaching its retention limit.
+ * - Id passed is incorrect
```

Comment:
`ID` is clearer

Reply:
That's my pref as well, but I was trying to go by this, which says either Id or identifier:

https://github.com/temporalio/documentation/blob/master/STYLE.md#abbreviation-of-identifier